src/components/credits-footer.js

```diff
@@ -29,22 +28,8 @@ function CreditsFooter({ doc, revision }) {
 
   const date = formatDate(request.timestamp);
 
-  const renderLongUrlText = urlText => {
-    const parts = urlText.replace(/[^\w]\b/g, c => `${c}${ZERO_WIDTH_SPACE}`).split(ZERO_WIDTH_SPACE);
-    return (
-      <Fragment>
-        {parts.map((part, index) => (
-          <Fragment key={index.toString()}>
-            {!!index && <wbr />}
-            {part}
-          </Fragment>
-        ))}
-      </Fragment>
-    );
-  };
-
-  const renderUrl = () => (<a rel="noopener noreferrer" target="_blank" href={url}>{renderLongUrlText(url)}</a>);
-  const renderOriginalUrl = () => (<a rel="noopener noreferrer" target="_blank" href={originalUrl}>{renderLongUrlText(originalUrl)}</a>);
+  const renderUrl = () => <LiteralUrlLink rel="noopener noreferrer" target="_blank" href={url} />;
+  const renderOriginalUrl = () => <LiteralUrlLink rel="noopener noreferrer" target="_blank" href={originalUrl} />;
```

Comment:
I guess the `rel` attribute could go inside the component, cause we always try to set it to "noopener noreferrer"

Reply:
It's a side effect of having `target="_blank"`, otherwise it wouldn't be there. But I created a `targetBlank` prop in the `LiteralUrlLink` component that puts either both or none there, so we do not have to specify it everywhere (we only use it so far with `target="_blank"` anyways).

Reply:
Ah true, good idea 👍